pages/wg/outreach.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: page
+title: Outreach Working Group
+subtitle:
+permalink: /wg/outreach/
+---
+
+# Overview
+
+The US-RSE Outreach Working Group is dedicated to broadening the US-RSE's
+engagement with the RSE and wider computational science and engineering (CSE)
+communities. We strive to raise awareness of the US-RSE as an association and
+RSE as a professional designation through engagement with industry, academia,
+national labs, and students.
+
+Those interested in contributing to this goal are encouraged to join our
+bi-weekly virtual meetings. To get involved, visit the
+[`#outreach`](https://usrse.slack.com/messages/outreach) channel on the US-RSE
+slack or contact the
+<a href="mailto:outreach_wg@us-rse.org">Outreach working group list</a>.
+
+# Work Products
+
+The US-RSE Outreach working group is new as of 2022. The following events were
+either completed before the creation of the official working group or are
+initiatives for future work.
+
+## Existing Efforts
+
+### RSE Stories
+
+Created by Vanessa Sochat, the [RSE Stories podcast](https://us-rse.org/rse-stories/)
+aims to cast a spotlight on the differing backgrounds and paths professionals
+have taken to become RSEs and help shape our understanding of what an RSE really is.
+
+The podcast has existed since 2019 and is looking for more volunteers both for
+interviewees and production assistance.
+
+### Conference and Workshop Participation
+
+The US-RSE has had a presence at several conferences and workshops throughout
+its lifetime. For example:
+
+- Supercomputing 2021: [RSE-in-HPC Workshop](https://us-rse.org/rse-hpc-2021/), Student Program Orientation
+- PEARC21: [RSE Career Panel](https://us-rse.org/2021-07-20-pearc21-panel/)
+- Supercomputing 2020: [RSE-in-HPC Workshop](https://us-rse.org/rse-hpc-2020/)
+- PEARC20: [Research Software Engineers Community Workshop](https://us-rse.org/events/2020/2020-07-pearc20/)
+- PEARC19: Building a Community of Research Software Engineers Birds of a Feather - [Proposal](https://ssl.linklings.net/conferences/pearc/pearc19_program/views/includes/files/bof111s1.pdf), [Event](https://us-rse.org/events/2019/pearc-2019/)
+- EDUCAUSE 2019: [Supporting Research Software and Research Software Engineers Poster](https://events.educause.edu/annual-conference/2019/agenda/supporting-research-software-development)
+
+### External Collaboration
+
+The US-RSE has collaborated with other organizations in the past for events
+such as the [Early Career Panel](https://academicdatascience.org/cdn/early-career-panel)
+([Academic Data Science Alliance](https://academicdatascience.org/)).
+
+## Future Directions
+
```

Comment:
I think it would be cool to see something here about national labs, or more generally getting institutions that are largely not recognizing the effort involved (primarily my employer). I had a long chat with some folks on our Mattermost channel yesterday sort of stamping my foot a bit - and the result was "well you should give a talk about US-RSE at X event." and that definitely is outreach!

Reply:
We had a conversation about "internal outreach" during our first meeting - as in, talking to our own employers just as much as we talk to everyone else. I'll add an "additional" section.